pages/common/ruby.md

```diff
@@ -0,0 +1,23 @@
+# ruby
+
+> Ruby programming language interpreter.
+
+- Open an Interactive Ruby Shell (REPL):
+
+`irb`
+
+- Execute a Ruby script:
+
+`ruby {{script.rb}}`
+
+- Execute a single command in the command line:
```

Comment:
This doesn't seem clear here. Could we change this to "a ruby command", maybe?

Reply:
Sure thing, I'll change it to "Execute a single Ruby command in the command line"

Reply:
Isn't it implicit that it will be a ruby command ? Seems pretty redundant to mention "ruby command" in a page for ruby.

Reply:
Hrm maybe, but `command` to me means a bash command or something. I don't know